package.json

```diff
@@ -17,7 +17,8 @@
   "resolutions": {
     "bl": "^1.2.3",
     "websocket-extensions": "^0.1.4",
-    "json-schema": "^0.4.0"
+    "json-schema": "^0.4.0",
+    "fast-json-patch": "^3.1.1"
```

Comment:
the vulnerability is coming from `fast-json-patch` version  3.0.0-1, which  `swagger-client` using, but there is no version of `swagger-client` that using the fixed version (3.1.1)